src/perfcollect/perfcollect

```diff
@@ -1327,6 +1341,11 @@ SetupLTTngSession()
         elif [ "$events" == "threading" ]
         then
             EnableLTTngEvents ${DotNETRuntime_ThreadingKeyword[@]}
+        elif [ "$events" == "kernel" ]
```

Comment:
Hmm... your comment here makes a good point.  Let's change this to "processlifetime" (and adjust the other names like "LTTng_Kernel_Default" to "Lttng_Kernel_ProcessLifetime") so that we don't define the kernel name.

Reply:
good point!